### ISSUE TITLE
ORCA screen reader is not announcing "Schemes" label name along with its options.: A11y_VS Code extension for API Center_Definition_OpenAPIDocumentation_ScreenReader

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/vscode-azureapicenter#112]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG1.3.1; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #ORCA; #Linux; #FTP; #Element:Dropdown; #External; #External:SwaggerUI; #eDAD:3p:220345; #ThirdParty;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Linux- Ubuntu 22.04.4
Screen Reader: ORCA

### Repro Steps:

1. Start ORCA screen reader.
2. Create an API center in azure portal.
3. Open Visual studio Code.
4. Tab till API center extension in the left side panel and press enter.
5. Tab till definition under the Api name which you have created, and press enter.
6. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
7. Go to "Open API Documentation" using the down arrow and press enter.
8. Demo conference API screen will appear. TAB to "Schemes" dropdown.
9. Observe the issue.

### Actual Result:
ORCA screen reader is not announcing "Schemes" label name along with its options, as label is not associated with its field.

Similar issue is also observed in below scenario:
Issue 1: Definition_ Open API Documentation

1. Start ORCA screen reader.
2. Create an API center in azure portal.
3. Open Visual studio Code.
4. Tab till API center extension in the left side panel and press enter.
5. Tab till definition under the Api name which you have created, and press enter.
6. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
7. Go to "Open API Documentation" using the down arrow and press enter.
8. Demo conference API screen will appear. TAB to "Authorize" button and hit ENTER key.
9. "Available Authorization" dialog will open. TAB to "Value" edit field and observe that label is not associated with its field.

### Expected Result:
ORCA screen reader should be announcing "Schemes" label name along with its options.

### User Impact:
Screen reader users will not get the information about the control and it purpose.

### Attachment:
![112_A11y_VS Code extension for API Center_Definition_OpenAPIDocumentation_ScreenReader](https://github.com/microsoft/vscode-azureapicenter/assets/90901519/ceb3fe2a-45e0-495f-9748-44a582759c9b)

![112_A11y_VS Code extension for API Center_Definition_OpenAPIDocumentation_Similar issue](https://github.com/microsoft/vscode-azureapicenter/assets/90901519/d0014f8c-c6c5-4030-805d-2f56114e8cf1)

**Comment by AnKushSingh05** [comment]
IRT Bug logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/220345

**Comment by Yash14j** [comment]
### GitHub Tags:
#External;#External:SwaggerUI:eDAD:3p:220345;#ThirdParty;

**Comment by AnKushSingh05** [comment]
External bug: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/220345

**Comment by AnKushSingh05** [comment]
External bug : https://github.com/swagger-api/swagger-ui/issues/9954

**Comment by kupatkar99** [comment]
GithubTags:#ResolvedByTT;
